sklearn/linear_model/passive_aggressive.py

```diff
@@ -110,6 +121,16 @@ def partial_fit(self, X, y, classes=None):
         -------
         self : returns an instance of self.
         """
+        if self.class_weight == 'balanced':
+            raise ValueError("class_weight 'balanced' is not supported for "
+                             "partial_fit. In order to use 'balanced' "
+                             "weights, from the sklearn.utils module use "
+                             "compute_class_weight('balanced', classes, y). "
```

Comment:
I'm concerned this can be misleading if the user's variables aren't actually called `classes` and `y`.

I suggest:
```
for 'balanced' weights, use `sklearn.utils.compute_class_weight` with `class_weight='balanced'
```
But this is obviously not a big deal.